portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro

```diff
@@ -608,6 +608,23 @@ definition {
 			value1 = "${pageNumber}");
 	}
 
+	macro viewAutocompleteSuggestions {
+		Type(
+			locator1 = "ACSegments#TEXT_INPUT",
+			value1 = "${textInput}");
```

Comment:
Hey @joshitagaki, I think you forgot to pass the value of the **index** that this path uses:

```
Type(
	index = "${index}",
	locator1 = "ACSegments#TEXT_INPUT",
	value1 = "${textInput}");
```

Reply:
I think you could create another macro responsible just for editing the field. I think this _viewAutocompleteSuggestions_ macro should only do checks and not actions as well. What do you think?